### PR TITLE
Make Vulcan Metrics send findings to DevHose

### PR DESCRIPTION
**Pull request by julianvilas**

### CODE REVIEW

File: stable/metrics/values.yaml [hunk 1]
```diff
@@ -31,14 +31,18 @@ conf:
   sqsPollingInterval: 10
   checksQueueArn: arn:aws:sqs:local:TBD:TBD
   scansQueueArn: arn:aws:sqs:local:TBD:TBD
+  findingsQueueArn: arn:aws:sqs:local:TBD:TBD
```
Comment (jesusfcr) [inline stable/metrics/values.yaml:34]: Please, declare the new findingsQueueArn in stable/vulcan/values.yaml:

-  In the metrics section
- In the goaws.yaml section

File: stable/metrics/templates/deployment.yaml [hunk 2]
```diff
@@ -54,10 +56,14 @@ spec:
...
             value: "{{ .Values.conf.devHoseTenant }}"
-          - name: LEDGER_SOURCE
-            value: "{{ .Values.conf.ledgerSource }}"
+          - name: DEVHOSE_METRICS_SOURCE
+            value: "{{ .Values.conf.devHoseMetricsSource }}"
+          - name: DEVHOSE_FINDINGS_SOURCE
+            value: "{{ .Values.conf.devHoseFindingsSource }}"
           - name: REDIS_ADDR
             value: "{{ .Values.conf.redisAddr | default (include "metricsRedisAddr" .)}}"
+          - name: VULCAN_API_BASE_URL
```
Comment (jesusfcr) [inline stable/metrics/templates/deployment.yaml:65]: I would change this and default to the calculated VULCAN_API internal endpoint, this way you don't have to set any value for that link.

You can see an example in the continuous component.

**Review by jesusfcr — CHANGES_REQUESTED** [review]
Some minor suggestions to integrate better with the internal infra and API components.

File: stable/metrics/values.yaml (hunk 1 above)
Comment (julianvilas) [reply stable/metrics/values.yaml:34]: It should be done now.

File: stable/metrics/templates/deployment.yaml (hunk 2 above)
Comment (julianvilas) [reply stable/metrics/templates/deployment.yaml]: It should be done now.

**Review by jesusfcr — APPROVED** [review]
LGTM

**Pull request merged**